src/app/template/template-tags/template-tags.component.ts

```diff
@@ -26,5 +27,13 @@ export class TemplateTagsComponent extends TagsComponent<BaseTemplateModel> {
   ) {
     super(dialogService, tagService);
   }
+
+  public ngOnChanges(changes: SimpleChanges) {
+    if (changes['entity']) {
+      this.resourceType = this.entity.isTemplate
```

Comment:
Please, do not use getter method of the model as we are going to get rid of models.
maybe `this.entity.resourceType || this.resourceType` can be used instead for saving